GoogleChromeLabs/ps-analysis-tool
# Enhancements: UI/UX changes to sidebar

## Description
This PR adds enhancements to the PSAT sidebar.
- Separate menu items by adding space and a divider.
- Move the settings menu item to the bottom.
- Open dropdowns programmatically if opened from the dashboard panel.
- Keep Cookie dropdown closed by default.
- Increase the menu item height for Dashboard, Wiki, Stories, and Settings.


## Relevant Technical Choices



## Testing Instructions
- Open PSAT and the sidebar should have a divider separating the section and settings item at the bottom.
- The cookie dropdown should be closed by default, height of Dashboard, Wiki, Stories and Settings should be increased.
- Move to the dashboard and open a nested panel, the dropdown should open.


## Additional Information:



## Screenshot/Screencast



---

## Checklist






Fixes #
